src/main/java/com/blog/config/MvcWebConfig.java

```diff
@@ -21,20 +38,59 @@
 })
 @ComponentScan("com.blog")
 @Import(BeansConfiguration.class)
-public class MvcWebConfig implements WebMvcConfigurer {
+public class MvcWebConfig implements WebMvcConfigurer, ApplicationContextAware {
 
     @Autowired
-    private ApplicationContext applicationContext;
+    ServletContext servletContext;
+    @Autowired
+    ApplicationContext applicationContext;
+
+    @Override
+    public void addFormatters(FormatterRegistry registry) {
+        registry.addConverter(getTagConverterToTag());
+        registry.addConverter(getTagConverterToString());
+        registry.addFormatter(new TagFormatter());
+    }
+
+    @Bean(name = "conversionService")
+    public ConversionService getConversionService() {
+        FormattingConversionServiceFactoryBean factoryBean = new FormattingConversionServiceFactoryBean();
+        Set<Formatter> formatters = new HashSet<>();
+        formatters.add(new TagFormatter());
```

Comment:
Looks like duplicate with previous method. Why is it impossible to autowire TagFormatter